### PR TITLE
COIN-1146 make info accept colors and add networkFees infoBox

### PR DESCRIPTION
**Pull request by henri-ly**
Not sure where i can post in but in https://ledgerhq.atlassian.net/browse/COIN-1146 you will find some screens

### CODE REVIEW

File: src/components/ValidateOnDevice.js [hunk 1]
```diff
@@ -202,6 +203,19 @@ export default function ValidateOnDevice({
           </View>
         </View>
       </ScrollView>
+      {mainAccount.currency.family === "polkadot" ? (
```
Comment (henri-ly) [inline src/components/ValidateOnDevice.js:206]: I don't like the way of doing this, we don't have anyway without a "refacto" to know if the coin doesn't display the network Fees on the device.
I just made it work on the scope of polkadot. But if you have a more elegant way of doing it, i'm in

**Review by haammar-ledger — APPROVED** [review]

**Review by haammar-ledger — APPROVED** [review]

File: src/locales/en/common.json [hunk 2]
```diff
@@ -3258,7 +3258,8 @@
           }
         }
       }
-    }
+    },
+    "networkFees": "Network fees are automatically set by the Polkadot consensus, you won't be able to review them on your device"
```
Comment (rjarasson-ledger) [inline src/locales/en/common.json:3262]: I would have put this at the beginning of the polkadot key (with the lockedBalance etc.. translation).But that's really a minor suggestion. Ignore if there is no other change to do.

File: src/locales/en/common.json [hunk 3]
```diff
@@ -3366,7 +3367,7 @@
     "validatorAddress": "Validator address",
     "rewardAmount": "Reward amount",
     "undelegatedAmount": "Undelegated amount",
-    "memo": "Memo"
+    "memo": "Memo",
```
Comment (rjarasson-ledger) [inline src/locales/en/common.json:3370]: I don't think it's ok.
JSON usually does not allow trailing comma.

File: src/locales/fr/common.json [hunk 4]
```diff
@@ -3376,7 +3377,7 @@
     "validatorAddress": "Validator address",
     "rewardAmount": "Reward amount",
     "undelegatedAmount": "Undelegated amount",
-    "memo": "Memo"
+    "memo": "Memo",
```
Comment (rjarasson-ledger) [inline src/locales/fr/common.json:3380]: no trailing comma.

**Review by rjarasson-ledger — CHANGES_REQUESTED** [review]

**Pull request closed without merging**